java/src/main/java/com/google/appengine/tools/mapreduce/RetryExecutor.java

```diff
@@ -0,0 +1,19 @@
+package com.google.appengine.tools.mapreduce;
+
+import com.github.rholder.retry.RetryException;
+import com.github.rholder.retry.RetryerBuilder;
+
+import java.util.concurrent.Callable;
+import java.util.concurrent.ExecutionException;
+
+public class RetryExecutor {
+
+  public static <V> V call(RetryerBuilder<V> retryerBuilder, Callable<V> callable) {
+    try {
+      return retryerBuilder.build().call(callable);
+    } catch (ExecutionException | RetryException e) {
+      throw new RuntimeException(e);
```

Comment:
prob unwrap `ExecutionException` at least, here write (`throw RuntimeException(ExecutionException::getCause)`, rather than `throw RuntimeException(ExecutionException(some Throwable)))`

Reply:
ok, makes sense